static/app/components/events/interfaces/performance/spanCountChart.tsx

```diff
@@ -0,0 +1,157 @@
+import {useState} from 'react';
+
+import {BarChart} from 'sentry/components/charts/barChart';
+import ErrorPanel from 'sentry/components/charts/errorPanel';
+import LoadingPanel from 'sentry/components/charts/loadingPanel';
+import {IconWarning} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {DateString, TagValue} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import SpanCountHistogramQuery from 'sentry/utils/performance/histogram/spanCountHistogramQuery';
+import {HistogramData} from 'sentry/utils/performance/histogram/types';
+import {formatHistogramData} from 'sentry/utils/performance/histogram/utils';
+import theme from 'sentry/utils/theme';
+
+export function SpanCountChart({issue, event, location, organization}: any) {
+  const transactionNameTag = event.tags.find(tag => tag.key === 'transaction');
```

Comment:
I'm seeing some duplicate code shared between `spanCountChart` and `durationChart`, later on we could either abstract the code into a reusable method for DRY or just use component specialization

Reply:
yup, saved that for later